public/locales/ru/translations.json

```diff
@@ -126,18 +126,18 @@
 
   "status title": "Статус",
   "of": "из",
-  "approve": "Необходимо подтвердить транзакцию ",
+  "approve": "Подтвердите транзакцию",
   "approved": "Транзакция подтверждена",
   "pending": "Транзакция в обработке",
-  "successful": "Транзакция успешна",
+  "successful": "Транзакция прошла успешно",
   "finishing": "Индексация данных списка",
   "finished": "Индексация завершена",
   "error": "Ошибка транзакции",
   "confirmation error": "Транзакция была отклонена",
-  "check wallet": "Проверьте ваш кошелек",
+  "check wallet": "Проверить кошелек",
   "block explorer": "Посмотреть в Block Explorer",
 
-  "leaderboard": "Таблица Лидеров",
+  "leaderboard": "Лидеры",
```

Comment:
Стоит учитывать размер элементов в контейнере навигатора. «Лидеры» туда поместятся. «Таблица Лидеров» тоже, но сделает эту область менее приятной для глаза. 

It is worth considering the size of the elements in the navigator container.«Лидеры» will fit there. «Таблица Лидеров» too, but it will make this area less pleasing to the eye.

Reply:
Много где вылазит текст. Им надо или размер или шрифт менять.
✅

Reply:
👍